modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/bulk_copy_entries.jsp

```diff
@@ -0,0 +1,35 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/document_library/init.jsp" %>
+
+<%
+DLCopyEntryDisplayContext dlCopyEntryDisplayContext = new DLCopyEntryDisplayContext(request, liferayPortletResponse, themeDisplay);
+
+dlCopyEntryDisplayContext.setViewAttributes();
+%>
+
+<div class="c-mt-3 sheet sheet-lg">
+	<react:component
+		module="document_library/js/DDMFolderSelector"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"copyActionURL", dlCopyEntryDisplayContext.getCopyEntriesActionURL()
+			).put(
+				"itemType", "fileEntry"
+			).put(
+				"redirect", dlCopyEntryDisplayContext.getRedirect()
+			).put(
+				"selectedEntries", dlCopyEntryDisplayContext.getSelectedEntries()
+			).put(
+				"selectionModalURL", dlCopyEntryDisplayContext.getSelectFolderURL()
+			).put(
+				"sourceFileName", dlCopyEntryDisplayContext.getSourceFolderName()
```

Comment:
needed?

Reply:
Yes, the sourceFolderName provides the name in the copy to page and the selectFolderURL allows the select folder modal to work.

Reply:
thanks!